WooCommerce/Classes/ViewRelated/Coupons/UsageDetails/CouponUsageDetails.swift

```diff
@@ -0,0 +1,173 @@
+                        Divider()
+                            .padding(.leading, Constants.margin)
+                            .padding(.leading, insets: geometry.safeAreaInsets)
+                        TitleAndTextFieldRow(title: Localization.usageLimitPerCoupon,
+                                             placeholder: Localization.unlimited,
+                                             text: $viewModel.usageLimitPerCoupon,
+                                             editable: false,
+                                             keyboardType: .asciiCapableNumberPad)
+                            .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        Divider()
+                            .padding(.leading, Constants.margin)
+                            .padding(.leading, insets: geometry.safeAreaInsets)
+                        TitleAndTextFieldRow(title: Localization.usageLimitPerUser,
+                                             placeholder: Localization.unlimited,
+                                             text: $viewModel.usageLimitPerUser,
+                                             editable: false,
+                                             keyboardType: .asciiCapableNumberPad)
+                            .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        Divider()
+                    }
+                    .background(Color(.listForeground))
+                    .padding(.bottom, Constants.margin)
+
+                    VStack(alignment: .leading, spacing: 0) {
+                        Divider()
+                        TitleAndTextFieldRow(title: Localization.limitUsageToXItems,
+                                             placeholder: Localization.allQualifyingInCart,
+                                             text: $viewModel.limitUsageToXItems,
+                                             editable: false,
+                                             keyboardType: .asciiCapableNumberPad)
+                            .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        Divider()
+                            .padding(.leading, Constants.margin)
+                            .padding(.leading, insets: geometry.safeAreaInsets)
+                        TitleAndValueRow(title: Localization.allowedEmails,
+                                         value: viewModel.allowedEmails.isNotEmpty ?
+                                            .content(viewModel.allowedEmails) :
+                                            .content(Localization.noRestrictions))
+                            .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        Divider()
+                    }
+                    .background(Color(.listForeground))
+                    .padding(.top, Constants.margin)
+
+                    ListHeaderView(text: Localization.usageLimits.uppercased(), alignment: .left)
```

Comment:
nit: do we also want to apply `.padding(.horizontal, insets: geometry.safeAreaInsets)` for landscape mode? (same for other `ListHeaderView` in the view)

<img width="882" alt="Screen Shot 2022-02-16 at 4 36 37 PM" src="https://user-images.githubusercontent.com/1945542/154226478-c5fc9c88-fa92-4d7c-ba6c-ba775d8d6f5c.png">

Reply:
Thanks for noticing! Fixed in e468581ce7afc2be4a8d4ee1744a6c8d7f212dce.